dataset_pack.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import yaml
+from pathlib import Path
+import os
+import shutil
+import cv2
+
+def clean(dataset_yaml_in, output_folder):
+    robot_names = [str(f).split('/')[-1] for f in Path(dataset_yaml_in).parent.iterdir() if f.is_dir()]
```

Comment:
not needed - the goal is to have a flat hierarchy, i.e., no subfolders in output folder